views/partials/_example.njk

```diff
@@ -35,6 +35,11 @@
     ```js
     {{ getNunjucksCode(examplePath) | safe }}
     ```
+  {%- if (params.group == 'components') %}
+    <p class="govuk-!-mt-r4 govuk-!-ml-r4">
+      <a href="https://github.com/alphagov/govuk-frontend/tree/master/packages/{{params.item}}/README.md#component-arguments" class="govuk-link"  target="_blank" data-track="link to table of arguments in Frontend readme">You can configure this component using the Nunjucks macro arguments</a>.
```

Comment:
According to the Trello ticket, only `Nunjucks macro arguments` should be a link:

```
'You can configure this component using the [Nunjucks macro arguments](URL for table of arguments).'
```

Reply:
yeah, my bad